app/scripts/lib/setupSentry.js

```diff
@@ -349,6 +349,19 @@ export const SENTRY_UI_STATE = {
     addSnapAccountEnabled: false,
     snapsAddSnapAccountModalDismissed: false,
     ///: END:ONLY_INCLUDE_IF
+    nameSources: {},
+    names: {
+      ethereumAddress: {},
+    },
+    preferences: {
+      petnamesEnabled: false,
```

Comment:
I guess we could just add this line in  `SENTRY_BACKGROUND_STATE` > `PreferencesController` > `preferences`

Reply:
@OGPoyraz this was added to make this test pass:
https://github.com/MetaMask/metamask-extension/blob/0c7aad2c81a000fec2378525d68d44cd562496c7/test/e2e/tests/errors.spec.js#L810